resources/views/airline.blade.php

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en" class="h-full">
+
+<head class="head">
+    @include('head')
+    @vite(['resources/js/airline.js'])
+</head>
+
+<header class="header">
+    @include('header')
+</header>
+
+<body class="h-full">
```

Comment:
Everything except `@vite(['resources/js/airline.js'])` and the body content should be inside a layout. Go [here](https://laravel.com/docs/10.x/blade#layouts-using-template-inheritance) to learn how to implement and use one.